modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc

```diff
@@ -93,22 +93,160 @@ image::odata-query-option-attributes.png["odataRequestAttributes is highlighted
+As OData shares some similarities with Java database connectivity (JDBC), the `transform-to-sql-select` operation takes advantage of such similarities to help the user create SQL queries for those resources. Then, the query is returned at the payload along with the query parameters at the attributes.
+
+=== How do I implement the transform to SQL select operation using APIkit for OData 4?
+
+<apikit-odata:transform-to-sql-select> is the name of the XML element that represents this operation, which has the following parameters: 
+
+|====
+| *Parameter name* | *Description*
+| Display name | This parameter configures the operation name.
+| Module configuration | Name of the related-module configuration.
+| Entity set name | The name of the entity set that you are trying to fetch #[attributes.entitySetName].
+| Page size | A number that defines the amount of pages in server-side-pagination.
+| Entity type name | A string obtained from the OData request using #[attributes.odataRequestAttributes], which is specified by the entity data model.
+| Entity type fields | An array of strings obtained from the OData request using #[attributes.odataRequestAttributes], which provides the list of property fields from the entity type.
+| Entity type keys | An object obtained from the OData request using #[attributes.odataRequestAttributes], which provides the map of entity key names and values.
+| System query options | Parameter obtained from the OData request using #[attributes.odataRequestAttributes].
+| Search transformation enabled | Parameter to define the behavior of the transformation.
+| Strict search term match | Parameter to define the behavior of the transformation.
+|====
+
+=== $filter expressions in OData 4 
+
+​​The `$filter` expression in OData uses comparison, logical, arithmetic, and grouping operators.
+
+==== Comparison Operators
+
+|====
+| *Operator* | *Description* | *Example* 
+| eq | Equal | Address/City eq 'Redmond'
+| ne | Not equal | Address/City ne 'London'
+| gt | Greater than | Price gt 20
+| ge | Greater than or equal to | Price ge 10
+| lt | Less than | Price lt 20
+| le | Less than or equal to | Price le 100
+| in | Is a member of | Address/City in ('Redmond', 'London')
+|====
+
+==== Logical Operators
+
+|====
+| *Operator* | *Description* | *Example*
+| and | Logical and | Price le 200 and Price gt 3.5
+| or | Logical or | Price le 3.5 or Price gt 200
+| not | Logical negation | not endswith(Description,'milk')
+|====
+
+==== Arithmetic Operators
+
+|====
+| *Operator* | *Description* | *Example*
+| add | Addition | Price add 5 gt 10
+| sub | Subtraction | Price sub 5 gt 10
+| mul | Multiplication | Price mul 2 gt 2000
+| div | Division | Price div 2 gt 4
+| mod | Modulo | Price mod 2 eq 0
+|====
+
+===== Grouping Operators
+
+|====
+| *Operator* | *Description* | *Example*
+| ( ) | Precedence grouping | (Price sub 5) gt 10
+|====
```

Comment:
I would get rid of this and instead point it out to the [ODATA spec](http://docs.oasis-open.org/odata/odata/v4.01/cs01/part1-protocol/odata-v4.01-cs01-part1-protocol.html#sec_BuiltinFilterOperations) in the below section